app/src/main/java/org/simple/clinic/contactpatient/ContactPatientUiRenderer.kt

```diff
@@ -31,6 +32,13 @@ class ContactPatientUiRenderer(
     }
   }
 
+  private fun renderCallResultOutcome(model: ContactPatientModel) {
+    if (model.hasCallResult) {
+      val callResult = model.callResult!!.get()
+      ui.showCallResult(callResult.outcome, callResult.timestamps.updatedAt)
+    }
```

Comment:
Will we raise another PR for hiding the call result?

Reply:
resolved